src/PyCall.jl

```diff
@@ -17,7 +17,7 @@ import Base: size, ndims, similar, copy, getindex, setindex!, stride,
        convert, pointer, summary, convert, show, haskey, keys, values,
        eltype, get, delete!, empty!, length, isempty, start, done,
        next, filter!, hash, splice!, pop!, ==, isequal, push!,
-       append!, insert!, prepend!, unsafe_convert
+       append!, insert!, prepend!, unsafe_convert, getproperty, propertynames
```

Comment:
I'd rather move towards the style of explicitly defining `Base.getproperty` etcetera, rather than importing.